src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTestBase.cs

```diff
@@ -40,7 +40,7 @@ public HttpClientHandlerTestBase(ITestOutputHelper output)
 #endif
             };
 
-        protected static HttpClient CreateHttpClient(string useVersionString) =>
+        protected HttpClient CreateHttpClient(string useVersionString) =>
```

Comment:
Some of these methods are static due to being run by `RemoteExecutor`. I don't remember exactly where, but I distinctly remember being bitten by this.
Are you sure this change doesn't break any existing inner and outer loop tests?

Reply:
Naively, I did only run this with innerloop WinHttpHandler tests. I see now that it breaks the SocketsHttpHandler tests, I will try an another approach I think.

Reply:
AFAK methods used with RemoteExecutor do not have to be static. But what ever state the code depends on needs to be marshaled expicitly as there is not magic how to serialize the object to different process.